idaloader.cpp

```diff
@@ -7,18 +7,52 @@
 // - fix XEX1 exports
 // - test!
 
-#include "../idaldr.h"
+//----------------------------------------------------------------------
+// Feb 2024 - Dwack - added external XML support using PugiXML
+// added idaxex.xml, based on xorlosers Xbox360.xml format
+//----------------------------------------------------------------------
+// 
+/** https://github.com/zeux/pugixml
+ * Copyright (c) 2006-2023 Arseny Kapoulkine
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following
+ * conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ */
+ //----------------------------------------------------------------------
+
+#include "idaldr.h"// my IDA SDK is setup differently, edit this to point to your header file location
```

Comment:
Again, if it worked before, I think a good rationale is required for changing this specifically for your setup. I assume your include directives are different; maybe update the readme?